cms/envs/common.py

```diff
@@ -2789,6 +2803,64 @@
     'PREPROCESSING_HOOKS': ['cms.lib.spectacular.cms_api_filter'],  # restrict spectacular to CMS API endpoints
 }
 
-#### Event bus publishing ####
-## Will be more filled out as part of https://github.com/edx/edx-arch-experiments/issues/381
-EVENT_BUS_PRODUCER_CONFIG = {}
+
+#### Event bus producing ####
```

Comment:
Annotations have been separated from the setting by an intervening function. But also I think we only need one copy of it?

Reply:
I went back and forth. I think it's useful to have it in both places but I don't feel too strongly

Reply:
[wontfix unless you want to push for it]